spec/models/adviser_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe Adviser, type: :model do
```

Comment:
Adviser was originally put in `models/` but then made better sense to have it in `lib/`, just looks like the spec file hasn't been updated accordingly.